src/cmd/cmd.cpp

```diff
@@ -467,14 +443,13 @@ int main(int argc, char **argv) {
 
     Cmd cmd;
     SyncJournalDb db(options.source_dir);
-    selectiveSyncFixup(&db, selectiveSyncList);
+    if (!selectiveSyncList.empty())
+        selectiveSyncFixup(&db, selectiveSyncList);
```

Comment:
Doesn't this change the behavior from 'no list means an empty list' to 'no list means: keep the previous list'?